cli_tools/osconfig_agent/osconfig_linux.go

```diff
@@ -79,8 +83,51 @@ func yumInstalls(pkgs []*osconfigpb.Package) error { return nil }
 
 func yumRemovals(pkgs []*osconfigpb.Package) error { return nil }
 
-func zypperRepositories(repos []*osconfigpb.ZypperRepository) error { return nil }
+func zypperRepositories(repos []*osconfigpb.ZypperRepository) error {
```

Comment:
We will need to add additional entries for at least a few things:
repo_gpgcheck
pkg_gpgcheck
gpgkey
and maybe 'type' since this is zypper

I'm fine with not adding in this PR but add a todo line for the gpg entries:
'// TODO: Write repo_gpgcheck, pkg_gpgcheck, and gpgkey entries.'